src/routes/about.svelte

```diff
@@ -8,25 +8,30 @@
 <CommonHero>
   <h1 slot="title">about</h1>
   <h2 slot="headline">
-    what is acm<span class="brand-blue">CSUF</span>
+    What is acm<span class="brand-blue">CSUF</span>

```

Comment:
This part doesn't have a question mark, which seems inconsistent with what's above.